internal/operators/manager.go

```diff
@@ -130,6 +131,7 @@ func (mgr *Manager) GenerateManifests(ctx context.Context, cluster *common.Clust
 		operator := mgr.olmOperators[clusterOperator.Name]
 		if operator != nil {
 			openshiftManifests, manifest, err := operator.GenerateManifests(cluster)
+			mgr.log.Infof("THIS IS CUSTOM BUILD OF SUBHAM %s", string(manifest))
```

Comment:
?

Reply:
> @subhamkrai @gamli75 can we please have a follow-up PR that reverts the changes in manager.go? Looks like that wasn't intended to be merged.

oops, I'll remove this in follow-up pr. Sorry

Reply:
https://github.com/openshift/assisted-service/pull/4920